src/vs/workbench/contrib/chat/browser/contrib/chatInputEditorContrib.ts

```diff
@@ -44,13 +54,21 @@ class InputEditorDecorations extends Disposable {
 		this.updateInputEditorDecorations();
 		this._register(this.widget.inputEditor.onDidChangeModelContent(() => this.updateInputEditorDecorations()));
 		this._register(this.widget.onDidChangeViewModel(() => this.updateInputEditorDecorations()));
+		this._register(this.chatService.onDidSubmitSlashCommand((e) => {
+			if (e.sessionId === this.widget.viewModel?.sessionId && !this._previouslyUsedSlashCommands.has(e.slashCommand)) {
+				this._previouslyUsedSlashCommands.add(e.slashCommand);
```

Comment:
Does this need to be removed when you ask some other question without that slash command? Or are you back in that mode even if you asked some other question in the meantime. Also does something need to happen when the chat session is cleared?

Reply:
> Or are you back in that mode even if you asked some other question in the meantime

Yes

> Also does something need to happen when the chat session is cleared?

Also yes